battleship.py

```diff
@@ -281,18 +240,18 @@ def did_anyone_win(p_board,c_board):
 			checkcell = letter + number
 			if p_board[checkcell] == "O":
 				o_count_player += 1
-			else:
-				fnord = "fnord"
+			# else:
+			# 	fnord = "fnord"
 			if c_board[checkcell] == "O":
 				o_count_computer += 1
-			else:
-				fnord = "fnord"
+			# else:
+			# 	fnord = "fnord"
 	if o_count_player == 0:
 		winner = "computer"
 	elif o_count_computer == 0:
 		winner = "player"
 	else:
-		winner = "null"
+		winner = None
```

Comment:
I changed how the game loop checks for a winner, so it's cleaner to return None than to return an in-place string.